pkg/securitycontext/provider.go

```diff
@@ -95,3 +96,53 @@ func makeCapabilites(capAdd []api.Capability, capDrop []api.Capability) ([]strin
 	}
 	return addCaps, dropCaps
 }
+
+func determineEffectiveSecurityContext(pod *api.Pod, container *api.Container) *api.SecurityContext {
```

Comment:
Do we need to have a hangout to work through this @smarterclayton @thockin ?  I feel like if we are still not clear on the semantics we need to get on the same page and update the recorded semantics in the proposal, since if there's any confusion, the proposal didn't successfully articulate things clearly.